conda-forge.yml

```diff
@@ -6,4 +10,5 @@ github:
   tooling_branch_name: main
 provider:
   linux_aarch64: default
-  linux_ppc64le: default
\ No newline at end of file
+  linux_ppc64le: default
+test_on_native_only: true
```

Comment:
Similarly this line disables the `test` phase of the recipe during cross-compilation as that involves running the command line, which won't work on the non-target architecture (as is the case in cross-compilation)